src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -153,6 +157,8 @@
             TokenTypes.LITERAL_FOR,
             TokenTypes.DO_WHILE,
             TokenTypes.ELLIPSIS,
+            TokenTypes.LITERAL_SWITCH,
```

Comment:
I can agree on lambda to be included in list of default tokens, but switch is not that obvious. I think we can exclude it from list of default tokens. Report shows that it is pretty common to do not put whitespace after switch.

@romani @rnveach @nmancus1 @pbludov what do you think?

Reply:
WhitespaceAround also contains `LITERAL_SWITCH`, we can put it in acceptable tokens if not in default.
https://github.com/checkstyle/checkstyle/blob/4b568976ada9210363162e64e232a0c67edbdc74/src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java#L527

Reply:
> Report shows that it is pretty common to do not put whitespace after switch.

I don't think it's intentional. For example, HBase uses `switch (` too: https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/916c493_2022100915/reports/diff/Hbase/xref/home/runner/work/checkstyle/checkstyle/contribution/checkstyle-tester/repositories/Hbase/hbase-examples/src/main/java/org/apache/hadoop/hbase/coprocessor/example/generated/BulkDeleteProtos.java.html#L98

The `switch` is the same statement as `for`, `while`, `catch`. I see no reason to make an exception for it.

Reply:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace demands space in similar structures, so I think it is ok to make is as default.

@Vyom-Yadav , please update inputs for Google style to make it clear our config will start to enforce space after such tokens.

Reply:
@pbludov , @rnveach please share your position on default

Reply:
> please update inputs for Google style to make it clear our config will start to enforce space after such tokens.

Done

Reply:
> @pbludov , @rnveach please share your position on default

SWITCH should be included by default.

Reply:
ok, thanks